guides/v2.1/frontend-dev-guide/themes/theme-structure.md

```diff
@@ -189,6 +189,15 @@ The directories and files structure described below is the most extended one. It
         View files that override the UI library files stored in <code>lib/web/css/source/lib</code>
       </td>
     </tr>
+    <tr>
```

Comment:
Hi @abrarpathan19. Thank you for the mentioning, but I have doubts about noting the framework parts here. The topic is about theme structure. `/lib/web/css/source/lib` is rather about frontend framework than a theme structure.

Reply:
> Hi @abrarpathan19. Thank you for the mentioning, but I have doubts about noting the framework parts here. The topic is about theme structure. `/lib/web/css/source/lib` is rather about frontend framework than a theme structure.

Hello @rogyar 
Is it possible the theme development without using these files `/lib/web/css/source/lib`?

That's why I have added this content 

Thank you !!!

Reply:
Hi @abrarpathan19. Thank you for the clarification. What I mean the main purpose of this topic is:

> This topic describes the file structure of a Magento theme

So, basically, it describes what files developer needs to create to implement a theme.
So, we should not go far from this point. Because we could say that it's not possible to develop a theme without many other components.

So the question that is answered is this topic is "What files do you need to bring with your theme to create a new working look and feel".

As a reference you can use https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/themes/theme-structure.html#theme-structure-comp

If you take a look at the directories three, we are referencing files that are located in <theme_dir> only.

All other approaches regarding a theme development are described in the separate topics, like this one https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/themes/theme-inherit.html which explains the fallback to `/lib/web/` resources and similar. 

I understand that It's a quite debatable topic, I just want to keep the things separated in the corresponding topics.

Thank you